scripts/upgrade

```diff
@@ -139,9 +139,26 @@ then
 	    sudo -u "$app" env PATH="$PATH" cargo build --features sqlite --release
 	popd
 
+    # Backup config file
+	tmpdir="$(mktemp -d)"
+	if [ -f $final_path/live/bitwarden_rs.env ]; then
+		cp -af "$final_path/live/bitwarden_rs.env" "$tmpdir/."
+	fi
```

Comment:
As there is no `bitwarden_rs.env` in the `$final_path/build/target/release/` maybe we don't need to backup/restore that file